app/src/main/res/drawable/updates_badge_circle_small.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="oval">
+
+  <solid android:color="@color/red" />
+
+  <stroke
+      android:width="1dp"
+      android:color="@color/default_background_color_light" />
```

Comment:
just checking.. will this be ok both themes?

Reply:
yes, i created a different drawable under the drawable-night folder